### ISSUE TITLE
infer shaped for node: MobilenetV1/MobilenetV1/Conv2d_0/Conv2D-MobilenetV1/MobilenetV1/Conv2d_0/BatchNorm/FusedBatchNorm-MobilenetV1/MobilenetV1/Conv2d_0/Relu6 op: Convolution failed

### ISSUE THREAD

**Issue by fathow666666** [OAID/Tengine#254]
First i use the mobilent model in Tengine model zoo,it is able to run.
Then I use the convert tool change tensorflow model to tengine model.   When use the mobilentv1.tmfile to classifay ,i says the error that   
infer shaped for node: MobilenetV1/MobilenetV1/Conv2d_0/Conv2D-MobilenetV1/MobilenetV1/Conv2d_0/BatchNorm/FusedBatchNorm-MobilenetV1/MobilenetV1/Conv2d_0/Relu6 op: Convolution failed
Prerun graph failed

![1](https://user-images.githubusercontent.com/63153189/78452125-09e2a900-76bc-11ea-8eb3-9945dde2d68a.png)
![2](https://user-images.githubusercontent.com/63153189/78452127-0cdd9980-76bc-11ea-9f57-be9f8dd49c68.png)
![3](https://user-images.githubusercontent.com/63153189/78452130-0fd88a00-76bc-11ea-97f8-b9b317a47dbb.png)

**Comment by liqi-c** [comment]
May you can try compare the differences between the two models？  through Netron tools .  
I will take a look at this problem .

**Comment by LEMON-LM** [comment]
The input tensor shape of tensorflow model is [?,224,224,3],it means NHWC,
but in the test code ,the  definition  of input tensor shape is [1,3,224,224],  it means NCHW , you  need change the test code.And try it again.

**Comment by liqi-c** [comment]
@fathow666666  the tensorflow model format is NHWC , but in our code default is nchw ,so you can modify like this : 

--- a/examples/classification.cpp
+++ b/examples/classification.cpp
@@ -67,7 +67,7 @@ bool run_tengine_library(const char* model_name, const char* tm_file, const char

     // set input
     int img_size = img_h * img_w * 3;
-    int dims[] = {1, 3, img_h, img_w};
+    int dims[] = {1, img_h, img_w ,3 };
